pipeline/merger/src/test/scala/uk/ac/wellcome/platform/merger/rules/ItemsRuleTest.scala

```diff
@@ -228,27 +229,33 @@ class ItemsRuleTest
     }
   }
 
-  it("Adds the METS item location to the Calm item") {
```

Comment:
I don't think METS items can be linked to Calm items in practice without an intermediate Sierra item, so I removed this test. If this were possible, you'd want some different behaviour – only using the METS item would lose the physical location.

Reply:
Agreed